details.php

```diff
@@ -25,6 +25,8 @@
 use mod_cpdlogbook\event\entry_viewed;
 
 require_once('../../config.php');
+require_once($CFG->libdir.'/formslib.php');
```

Comment:
Are the `require_once($CFG->libdir.'/formslib.php')` required? I don't think that most of these files need this, since only the form classes require this file. It seems like you might've copied both of the `require_once` calls from the `edit_entry.php` files and used that in all the files where the file manager is being used.

Reply:
If I recall correctly I was getting some errors and adding those in seemed to fix them, don't remember exactly what, something about formslib also requiring filelib. I'll check if it does need them still. I forgot to remove the debug line saying "there are attachments" anyway.

Reply:
Removed them and seems to work fine now